.circleci/config.yml

```diff
@@ -256,26 +317,40 @@ workflows:
       - release-nnoir:
           context:
             - docker-hub-creds
-          filters:
+          filters: &release-filter
```

Comment:
I think something like `nnoir-release-filter` is better because this name conflicts with the label below. (for readability)

Reply:
fixed